src/topology/algebra/group.lean

```diff
@@ -78,6 +78,9 @@ protected def homeomorph.mul_right (a : G) :
   continuous_inv_fun := continuous_id.mul continuous_const,
   .. equiv.mul_right a }
 
+@[simp, to_additive]
+lemma homeomorph.coe_mul_right (a : G) : ⇑(homeomorph.mul_right a) = λ g, g * a := rfl
```

Comment:
Can you add `mul_right_symm` here too to match `mul_left_symm` above? It seems that's the last gap between the left and right API.